content/projects/quintype.md

```diff
@@ -0,0 +1,44 @@
+---
++title: Quintype
++homepage: https://www.quintype.com/
++twitter: quintype_inc
++opensource: "No"
++typeofcms: "API Driven"
++supportedgenerators:
++  All
```

Comment:
This needs to be a list:

```yaml
supportedgenerators:
  - All
```

Reply:
@erquhart Hey fixed the issues. Can you merge this PR now?